docs/how_to/namespaced_apphooks.rst

```diff
@@ -1,45 +1,148 @@
+
+Once set up, an apphook configuration can be applied to to an apphook instance, in the *Advanced settings* of the page
+the apphook instance belongs to:
+
+.. image:: /how_to/images/select_apphook_configuration.png
+   :alt: selecting an apphook configuration application
+   :width: 400
+   :align: center
+
+The configuration is then loaded in the application's views for that namespace, and will be used to determined how it
+behaves.
+
+Creating an application configuration in fact creates an apphook instance namespace. Once created, the namespace of a
+configuration cannot be changed - if a different namespace is required, a new configuration will also need to be
+created.
+
 
-The concept of apphook configuration is to store all the configuration in an
-applications-specific model, and let the developer specify the desired options
-in a form. In the views the configuration model instance specific for the
-current application namespace is loaded (through a mixin) and thus it is
-available in the view to provide the configuration for the current namespace.
+********************************
+An example apphook configuration
+********************************
 
-Namespaces can be created on the fly in the Page admin Advanced settings.
+In order to illustrate how this all works, we'll create a new FAQ application, that provides a simple list
+of questions and answers, together with an apphook class and an apphook configuration model that allows it to
+exist in multiple places on the site in multiple configurations.
 
-When creating an application configuration, you are in fact defining a
-namespace, which is saved in the same field in the Page model as the
-plain namespaces.
+We'll assume that you have a working django CMS project running already.
 
+Using helper applications
+=========================
 
-step-by-step implementation
-###########################
+We'll use a couple of simple helper applications for this example, just to make our work easier.
 
-We're going to create a new application called FAQ. It is a simple list
-of Frequently asked questions. And we'll make it possible to setup multiple
-sets of FAQ Entries at different locations of the page tree, each with its
-individual set of entries.
 
-Lets create our new FAQ app:
+Aldryn Apphooks Config
+----------------------
+
+`Aldryn Apphooks Config <https://github.com/aldryn/aldryn-apphooks-config>`_ is a helper application that makes it
+easier to develop configurable apphooks. For example, it provides an ``AppHookConfig`` for you to subclass, and other
+useful components to save you time.
+
+In this example, we'll use Aldryn Apphooks Config, as we recommend it. However, you don't have to; if you prefer to can
+build the code you require by hand.
+
+Use ``pip install aldryn-apphooks-config`` in the usual way. Then, add ``aldryn_apphooks_config`` to your
+``INSTALLED_APPS`` in settings.
+
+
+Django AppData
+--------------
```

Comment:
This is a dependency of Aldryn Apphooks Config and it probably doesn't really need to be mentioned at all.